orcid-web/src/main/resources/freemarker/layout/public-layout.ftl

```diff
@@ -513,8 +513,9 @@
         '../common/maintenance_header.ftl'/> <#-- wtf --> <#nested>
     </div>
 </div>
+
 <!-- .container -->
-<#if RequestParameters['newFooter']??>
+<#if RequestParameters['NEW_FOOTER']?? || FEATURE['NEW_FOOTER']>
```

Comment:
1. We should make a easy to use tag.
2. Also the logic should be "if request parameter exist use it's value otherwise use db value".